service/matching/pollerHistory.go

```diff
@@ -85,12 +81,20 @@ func (pollers *pollerHistory) getPollerInfo(earliestAccessTime time.Time) []*tas
 		lastAccessTime := entry.CreateTime()
 		if earliestAccessTime.Before(lastAccessTime) {
 			result = append(result, &taskqueuepb.PollerInfo{
-				Identity:       string(key),
-				LastAccessTime: &lastAccessTime,
-				RatePerSecond:  value.ratePerSecond,
+				Identity:                  string(key),
+				LastAccessTime:            &lastAccessTime,
+				RatePerSecond:             defaultValue(value.ratePerSecond, defaultTaskDispatchRPS),
+				WorkerVersionCapabilities: value.workerVersionCapabilities,
 			})
 		}
 	}
 
 	return result
 }
+
+func defaultValue[T any, P ~*T](p P, def T) T {
```

Comment:
_laughs in [Default](https://doc.rust-lang.org/std/default/trait.Default.html)_

Reply:
Yes, it is different, but it allows this https://doc.rust-lang.org/std/option/enum.Option.html#method.unwrap_or_default to work automatically, which would be the closest equivalent